include/irr/asset/filters/CMatchedSizeInOutImageFilterCommon.h

```diff
@@ -14,25 +14,23 @@ namespace irr
 namespace asset
 {
 
+//! Base class for common matched size in-out images
 /*
-	Common filter for images where input and desired output
-	image data is known. The filter can execute various
-	converting actions on input image to get an output
-	image that will be a converted and ready to use one.
+	Common base class for filters for images where input 
+	and desired output image data is known. The filter 
+	can execute various converting actions on input image 
```

Comment:
The filter can -> The filters derived from it


Also as the name says, the input range is the same size as output this is why theres only one extent field